### PR TITLE
Add 2 images from idr0048

### PR DESCRIPTION
**Pull request by will-moore**
- As requested at https://forum.image.sc/t/converting-other-idr-images-into-public-zarr/44025/13
- 9846151.zarr was generated via bioformats2raw
- 9846152.zarr was generated via omero-cli-zarr, using PR: https://github.com/ome/omero-cli-zarr/pull/134

Inspect images at https://ome.github.io/ome-ngff-validator/?source=https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0048A/9846151.zarr/
and
https://ome.github.io/ome-ngff-validator/?source=https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0048A/9846152.zarr/

### CODE REVIEW

**Comment by will-moore** [comment]
cc @sbesson I don't seem to be able to add you as a reviewer?

File: _data/table.csv [hunk 1]
```diff
@@ -87,3 +87,5 @@ OME-NGFF version,EMBL-EBI bucket (current),SizeX,SizeY,SizeZ,SizeC,SizeT,Axes,We
 0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0101A/13457227.zarr,2048,2048,35,4,18,XYZCT,,,,CC BY 4.0,idr0101,,2022-10-13,13457227
 0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0101A/13457537.zarr,198,223,12,6,18,XYZCT,,,coordinateTransformation (translation) on dataset,CC BY 4.0,idr0101,,2022-10-13,13457537
 0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0101A/13457539.zarr,190,189,11,6,18,XYZCT,,,coordinateTransformation (translation) on multiscales,CC BY 4.0,idr0101,,2022-10-13,13457539
+0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0048A/9846151.zarr/,2947,5192,1402,3,1,XYZCT,,,bioformats2raw.layout,CC BY 4.0,idr0048,,2023-01-12,9846151
+0.4,https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0048A/9846152.zarr/,19120,13350,91,3,1,XYZCT,,,bioformats2raw.layout,CC BY 4.0,idr0048,,2023-01-12,9846152
```
Comment (sbesson) [inline _data/table.csv:91]: Isn't `bioformats2raw.layout`  incorrect as per the description of the PR?

**Review by sbesson — CHANGES_REQUESTED** [review]
I am confused by the `9846152.zarr` samples:

- the top-level https://ome.github.io/ome-ngff-validator/?source=https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0048A/9846152.zarr/ reports a multiscales dataset create by omero-cli-zarr
- https://ome.github.io/ome-ngff-validator/?source=https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0048A/9846152.zarr/0 reports another multiscale created by Bio-Formats
- https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0048A/9846152.zarr/OME/METADATA.ome.xml contains some metadata

I would postulate something unexpected happened during the generation

File: _data/table.csv (hunk 1 above)
Comment (will-moore) [reply _data/table.csv:91]: Ah, yes - I opened this PR when I started running the `bioformats2raw` conversion on the 2nd image, but then it failed and I used `omero-cli-zarr` instead. I'll fix...

**Comment by sbesson** [comment]
```
$ aws s3 ls --endpoint https://uk1s3.embassy.ebi.ac.uk s3://idr/zarr/v0.4/idr0048A/9846152.zarr/0/
                           PRE 0/
                           PRE 1/
                           PRE 2/
2023-02-02 11:55:05        422 .zarray
2023-01-18 13:14:29       2783 .zattrs
2023-01-18 13:14:29         23 .zgroup
```

Looks like objects were created an uploaded at very different times. If you still have the source of truth for the latest generated version of the daataset, my suggestion would either be to delete and re-upload or running `aws sync`.
In both cases, please do run a dry-run as deleting data from the bucket is pretty irreversible.

**Comment by will-moore** [comment]
I still have a copy of everything on `pilot-zarr1-dev`, and it doesn't have the extra files:
```
$ ls -alh 9846152.zarr/0
total 16K
drwxrwxr-x.  5 wmoore wmoore   68 Feb  1 12:03 .
drwxrwxr-x. 11 wmoore wmoore  165 Feb  2 11:51 ..
drwxrwxr-x. 93 wmoore wmoore 4.0K Feb  1 06:21 0
drwxrwxr-x. 93 wmoore wmoore 4.0K Feb  1 12:02 1
drwxrwxr-x. 93 wmoore wmoore 4.0K Feb  1 20:08 2
-rw-rw-r--.  1 wmoore wmoore  422 Feb  1 06:04 .zarray
```
I may have assumed after running `bioformats2raw` on that image the first time (2023-01-18) that it had completed, and I possibly tried uploading then, before realising that it had failed.

This was my upload command:

```
$ /home/wmoore/mc cp -r 9846152.zarr/ uk1s3/idr/zarr/v0.4/idr0048A/9846152.zarr/
...r/9846152.zarr/8/2/90/0/0: 62.54 GiB / 62.54 GiB ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 102.89 MiB/s 10m22s(
```

Just to double-check (I'll use `--fake` to test this) - do these look OK?

```
/home/wmoore/mc rm --fake uk1s3/idr/zarr/v0.4/idr0048A/9846152.zarr/0/.zattrs
/home/wmoore/mc rm --fake uk1s3/idr/zarr/v0.4/idr0048A/9846152.zarr/OME/METADATA.ome.xml
/home/wmoore/mc rm --fake uk1s3/idr/zarr/v0.4/idr0048A/9846152.zarr/0/0/.zarray
```

**Comment by will-moore** [comment]
@sbesson Happy for me to go ahead with these removals for real (without `--fake`)?

```
$ /home/wmoore/mc rm --fake uk1s3/idr/zarr/v0.4/idr0048A/9846152.zarr/0/.zattrs
DRYRUN: Removing  /idr/zarr/v0.4/idr0048A/9846152.zarr/0/.zattrs
$ /home/wmoore/mc rm --fake uk1s3/idr/zarr/v0.4/idr0048A/9846152.zarr/OME/METADATA.ome.xml
DRYRUN: Removing  /idr/zarr/v0.4/idr0048A/9846152.zarr/OME/METADATA.ome.xml
$ /home/wmoore/mc rm --fake uk1s3/idr/zarr/v0.4/idr0048A/9846152.zarr/0/0/.zarray
DRYRUN: Removing  /idr/zarr/v0.4/idr0048A/9846152.zarr/0/0/.zarray
```

Any other changes needed for this PR?

**Comment by sbesson** [comment]
> Any other changes needed for this PR?

I cannot comment on whether there are other files left from the former upload. The two additional rows in the table now look good to me.

**Comment by will-moore** [comment]
@sbesson You OK to update your "requested changes" status for this PR?
(I'll also go ahead and delete those 3 files...)

**Review by sbesson — APPROVED** [review]

**Pull request merged**